BeerOverflowWindowsApp/MainWindow.cs

```diff
@@ -91,33 +91,27 @@ private void GoButton_Click(object sender, EventArgs e)
                 var providerCount = providerList.Count;
                 var progressStep = 100 / providerCount;
                 var result = new BarDataModel();
-                
                 var currentProgressValue = 0;
                 GoButton.Enabled = false;
                 InitiateProgressBars();
                 UpdateProgressBars(currentProgressValue);
-
                 foreach (IBeerable provider in providerList)
                 {
                     CollectBarsFromProvider(provider, result, latitude, longitude, radius);
                     currentProgressValue += progressStep;
                     UpdateProgressBars(currentProgressValue);
                 }
-
                 HideProgressBars();
-
                 result.ForEach(bar => bar.BarId = bar.Title); // Temporary solution until we decide on BarId 
-                
                 // Display
                 result.GetRatings();
                 _barRating.BarsData = result;
-
                 var currentLocation = GetCurrentLocation();
                 foreach(var bar in _barRating.BarsData)
                 {
                     bar.DistanceToCurrentLocation = currentLocation.GetDistanceTo(new GeoCoordinate(bar.Latitude, bar.Longitude));
                 }
-                SortList(CompareType.Distance, SortOrder.Ascending);
+                SortList(CompareType.Distance);
```

Comment:
Not sure about this line being changed. I know the SortOrder.Ascending is redundant, but it could give the reader an understanding of what the code does? Or maybe the reader should know that sorting is, by default, ascending?

Reply:
SortList method already has this "SortOrder sortOrder = SortOrder.Ascending"